<launch>
	<node pkg="roborodney" name="laserPublisher" type="laserPublisher" />
	<node pkg="roborodney" name="bno055" type="bno055"/>
	<node pkg="roborodney" name="robotDriver" type="robotDriver" />
	<node pkg="roborodney" name="servoDriver" type="servoDriver" />
	<node pkg="roborodney" name="surroundingsScan" type="surroundingsScan" />
	<node pkg="roborodney" name="pictureScan" type="pictureScan" />
	<node pkg="roborodney" name="trayLeftRight" type="trayLeftRight" />
	<node pkg="roborodney" name="driveAuto" type="driveAuto" output="screen"/>
</launch>
